debian/changelog

```diff
@@ -1,29 +1,51 @@
 authd (0.3) noble; urgency=medium
 
+  * New upstream release
```

Comment:
Sorry I had updated the comment, maybe better to say

```suggestion
  * New major release
```

Or something similar? Being a native package there's not really an upstream :)

Reply:
Could we please stop bikeshedding on this kind of details and focus on the deliverables before FF. Everyone's time is precious.